vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp

```diff
@@ -74,38 +84,85 @@ $( document ).ready(function() {
 	
 	$("#createSpaceLinkBtn").click(function(e) {
 		var payload = {};
+		var posX = $("#bgImage").position().left;
+		var posY = $("#bgImage").position().top;
 		payload["x"] = storeX;
 		payload["y"] = storeY;
 		payload["rotation"] = $("#spaceLinkRotation").val();
 		payload["linkedSpace"] = $("#linkedSpace").val();
+		payload["spaceLinkLabel"] = $("#spaceLinkLabel").val();
 		payload["type"] = $("#type").val();
-		$("#arrow").remove();
+
 		$.post("<c:url value="/staff/space/${space.id}/spacelink?${_csrf.parameterName}=${_csrf.token}" />", payload, function(data) {
 			// TODO: show success/error message
 		});
-		$("#bgImage").on("click", function(e){});
-		$("#createSpaceLinkAlert").hide();	
-	});
-	
-	$('#spaceLinkRotation').change(function() {
-		$('#arrow').css('transform', 'rotate(' +$('#spaceLinkRotation').val()+ 'deg)');
+		$("#createSpaceLinkAlert").hide();
+		$("#label-visibility").attr("id","");
+		$("#link").attr("id","");
+		$("#icon").attr("id","");
+		show(payload);		
 	});
 		
 	$('#changeBgImgButton').click(function(file) {
 		$("#createSpaceLinkAlert").hide();
-		$("#changeBgImgAlert").show();	
-		
+		$("#changeBgImgAlert").show();			
 	});
 			
 	$("#changeBgImgAlert").draggable();
 		
 	$('#spaceLinkCreationModal.draggable>.modal-dialog>.modal-content>.modal-header').css('cursor', 'move');
 	
 	$('#spaceLinkRotation').change(function() {
-		$('#arrow').css('transform', 'rotate(' +$('#spaceLinkRotation').val()+ 'deg)');
+		$('#link').css('transform', 'rotate(' +$('#spaceLinkRotation').val()+ 'deg)');
 	});
 	
+	$(".target").change(function() {
+		var spaceLink = {};
+		spaceLink["x"] = storeX;
+		spaceLink["y"] = storeY;
+		spaceLink["spaceLinkLabel"] = $("#spaceLinkLabel").val();
+		spaceLink["type"] = $("#type").val();
+		show(spaceLink);
+	});
+
+	function show(spaceLinks) {
```

Comment:
spaceLinks has only data for one link, right? so it should not be 'Links' but 'Link'